src/psyclone/tests/psyir/nodes/call_test.py

```diff
@@ -628,6 +629,539 @@ def test_call_get_callees_local(fortran_reader):
     assert result == [psyir.walk(Routine)[1]]
 
 
+def test_call_get_callee_1_simple_match(fortran_reader):
+    '''
+    Check that right routine has been found.
```

Comment:
"...found for a single routine implementation."

Reply:
Changed to "Check that the right routine has been found for a single routine implementation."